app/assets/src/components/views/amr_heatmap/amr_heatmap_vis.scss

```diff
@@ -0,0 +1,19 @@
+// @import "~styles/themes/colors";
+
+.AMRHeatmapContainer {
+  // margin-top: 100px;
+  overflow-x: auto;
+  margin-bottom: 300px;
+
+  svg {
+    display: block;
+    margin: 0 auto;
+  }
+}
+
+.AMRHeatmapVis {
+}
+
+.tabs {
```

Comment:
Just to make sure: Jenn's mocks had tabs (Taxon, AMR), but you can just do a separate page for now. Having two tabs complicates some of the logic around saving/sharing visualizations.